src/webviews/apps/plus/graph/GraphWrapper.tsx

```diff
@@ -1416,15 +1418,6 @@ export function GraphWrapper({
 				) : (
 					<p>No repository is selected</p>
 				)}
-				<button
-					className="column-button"
-					type="button"
-					role="button"
-					data-vscode-context={context?.settings || JSON.stringify({ webviewItem: 'gitlens:graph:settings' })}
-					onClick={handleToggleColumnSettings}
-				>
-					<span className="codicon codicon-settings-gear" aria-label="Column Settings"></span>
-				</button>
```

Comment:
This was removed, but how can GitLens provide the `data-vscode-context` now? @ericf-axosoft @axosoft-ramint

Reply:
The graph sets the `data-vscode-context` for the button by using the actual `contexts.settings`.

Reply:
Yep, we pass `context` to the graph as a prop `contexts` and `context.settings` makes it to the button via the prop as `contexts.settings`